src/components/organisms/CurrencySelector/CurrencySelector.jsx

```diff
@@ -14,7 +14,7 @@ function CurrencySelector() {
           <CurrencyItem
             flag={flagMX}
             country="México"
-            currency="MXN peso"
+            currency="MXN pesos"
```

Comment:
I think that could be great have a list of currencies or a JSON with all currencies that we want support

Reply:
Applied, thanks Felipe